OpenRA.Game/Game.cs

```diff
@@ -172,6 +172,38 @@ internal static void StartGame(string mapUID, WorldType type)
 			GC.Collect();
 		}
 
+		public static void RestartGame()
```

Comment:
The plan is for RestartGame to possibly do more in the future but right now it restarts missions and replays. As for moving them together, I'm not exactly sure how I would do that.